utils/misc.ml

```diff
@@ -881,8 +881,80 @@ let print_if ppf flag printer arg =
   if !flag then Format.fprintf ppf "%a@." printer arg;
   arg
 
+
 type filepath = string
 type modname = string
 type crcs = (modname * Digest.t option) list
 
 type alerts = string Stdlib.String.Map.t
+
+
+module EnvLazy = struct
```

Comment:
Do we really want to grow misc? Couldn't `EnvLazy` be in its own file?

Reply:
I would rather grow misc than risking to introduce module name conflicts. We can always revisit this later.